doc/sphinx/language/core/inductive.rst

```diff
@@ -17,13 +17,14 @@ Inductive types
       constructor ::= @ident {* @binder } {? @of_type }
 
    This command defines one or more
-   inductive types and its constructors.  Coq generates destructors
+   inductive types and its constructors.  Coq generates
+   :term:`induction principles <induction principle>`
```

Comment:
IMO the `:term:` and `:gdef:` here should be inverted. It doesn't seem reasonable that `:gdef:` appears inside an example instead of here.